ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/category/CategoryLabelProvider.java

```diff
@@ -107,6 +107,10 @@ public void dispose() {
 			fSiteFeatureImage.dispose();
 			fSiteFeatureImage = null;
 		}
+		if ((fSiteBundleImage != null) && (fSiteBundleImage.isDisposed() == false)) {
```

Comment:
```suggestion
		if ((fSiteBundleImage != null)) {
```
disposing a disposed image is a noop so I don't se a reason to add this strange check here